source/controllers/asd.py

```diff
@@ -52,6 +52,14 @@ def create_asd(disk):
         if disk.state == 'MISSING':
             raise RuntimeError('Cannot create an ASD on missing disk {0}'.format(disk.name))
 
+        ipaddresses = Configuration.get('{0}/network|ips'.format(ASDController.CONFIG_ROOT))
+        if len(ipaddresses) == 0:
+            ipaddresses = ASDController._local_client.run("ip a | grep 'inet ' | sed 's/\s\s*/ /g' | cut -d ' ' -f 3 | cut -d '/' -f 1", allow_insecure=True).strip().splitlines()
+            ipaddresses = [found_ip.strip() for found_ip in ipaddresses if not found_ip.strip().startswith('127.')]
+            if len(ipaddresses) == 0:
+                raise RuntimeError('Could not find any IP on the local node')
+            ipaddresses = [ipaddresses[0]]  # We make list of first IP found on system if None were configured during initial setup
```

Comment:
Currently a WIP to register with all ips. For the initial version this is fine but it has to be adjusted for the final one.